ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-isis.j2

```diff
@@ -12,15 +12,38 @@
 {%     if router_isis.net is defined %}
 | Net-ID | {{ router_isis.net }} |
 {%     endif %}
+{%     if router_isis.router_id is arista.avd.defined %}
+| Router-ID | {{ router_isis.router_id }} |
+{%     endif %}
 {%     if router_isis.is_type is defined %}
 | Type | {{ router_isis.is_type }} |
 {%     endif %}
-{%     if router_isis.address_family is defined %}
```

Comment:
keep this code for compatiblity.

Reply:
It is there but moved to better reflect config order.